parser.cpp

```diff
@@ -1185,9 +1194,9 @@ namespace Sass {
     { return new (ctx.mem) Null(pstate); }
 
     if (lex< identifier >()) {
-      String_Constant* str = new (ctx.mem) String_Constant(pstate, lexed);
+      String_Constant* str = new (ctx.mem) String_Quoted(pstate, lexed);
```

Comment:
In the end `String_Quoted` will contain an unquoted value (it means the parsed string might has been a quoted string), while `String_Constant` is taken as is.